addons/mrp/views/ir_attachment_view.xml

```diff
@@ -14,9 +14,12 @@
                 <field name="create_uid"/>
                 <templates>
                     <t t-name="kanban-box">
+                        <t t-set="fileRequest" t-value="record.type.raw_value === 'empty'"/>
                         <div class="oe_kanban_global_area o_kanban_attachment oe_kanban_global_click">
                             <div class="o_kanban_image">
-                                <div class="o_kanban_image_wrapper">
+                                <t t-set="binaryPreviewable"
+                                   t-value="new RegExp('(image|video|application/pdf|text)').test(record.mimetype.value) &amp;&amp; record.type.raw_value === 'binary'"/>
+                                <div t-attf-class="o_kanban_image_wrapper #{(webimage or binaryPreviewable) ? 'o_kanban_previewer' : ''}" t-att-title="fileRequest ? 'Upload file' : ''">
```

Comment:
@anr-odoo Why this change ? Can we have a file request with mrp documents ? 
Also, please don't push anything from now, I rework your branch to merge it (docstring, guidelines, ...).

Reply:
@svs-odoo  By mistake change was there something gets messed in rebasing.